charts/harvester-pcidevices-controller/values.yaml

```diff
@@ -0,0 +1,54 @@
+# Default values for harvester-pcidevices-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: rancher/harvester-pcidevices
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: master-head
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  privileged: true
+
+service:
+  type: ClusterIP
+  port: 80
+
+
+resources:
+  limits:
```

Comment:
The limits are quite tight. Maybe increasing it a little bit?

Reply:
it is currently running with these limits. I can bump it up, but since it will be a default install i don't want to make harvester too bloated.

Reply:
I measured the pcidevices controller averaging about 20MB ram so 100 should be more than enough